CHANGELOG.md

```diff
@@ -11,6 +11,7 @@
 
 * **[Fix]** Avoid opening browser to check for sign-in information after receiving an SSL error while checking for app updates (which often happens when using a public WIFI).
 * **[Fix]** When in-app update permissions become invalid and need to open browser again, updates are no longer postponed after sign-in (if user previously selected the action to postpone for a day).
+* **[Fix]** Fix a possible deadlock when activity resumes during background operation for `Distribute.isEnabled()` call.
```

Comment:
Isn't setEnabled or notifyUserConfirmation also impacted? Should be more generic in that case.

Reply:
`notifyUserConfirmation` calls `isInstanceEnabledAsync` internally, so it's the same.

Reply:
```suggestion
* **[Fix]** Fix a possible deadlock when activity resumes during background operation for some `Distribute` public APIs like `Distribute.isEnabled()`.
```